packages/cactus-plugin-ledger-connector-sawtooth-socketio/package.json

```diff
@@ -11,23 +11,19 @@
     "init-sawtooth": "cp -af ../../yarn.lock ./dist/yarn.lock"
   },
   "dependencies": {
-    "@types/node": "14.17.32",
-    "body-parser": "1.17.2",
-    "cbor": "6.0.1",
-    "config": "3.3.7",
-    "cookie-parser": "1.4.6",
-    "debug": "4.1.1",
-    "express": "4.15.5",
-    "js-yaml": "3.14.1",
-    "jsonwebtoken": "8.5.1",
-    "log4js": "6.4.1",
-    "morgan": "1.10.0",
-    "serve-favicon": "2.4.5",
-    "shelljs": "0.8.5",
-    "socket.io": "4.4.1",
-    "xmlhttprequest": "1.8.0"
```

Comment:
@micoferdinand98 Was the change on this line intended or did the conflict resolution go sideways?

Reply:
the resolution go sideways.

Reply:
the change happened when I squash other commits and resolved the merge conflict